src/configs/latest.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2016 Palantir Technologies, Inc.
+ * Copyright 2018 Palantir Technologies, Inc.
```

Comment:
we are not in the habit of updating copyright years in existing files, only in new files -- needless churn. they still apply in the present. the top one on the docs site makes sense though.